pycytominer/cyto_utils/ingest_config.ini

```diff
@@ -0,0 +1,11 @@
+[filenames]
```

Comment:
i think it makes sense to move this file inside a `database_config/` folder within cyto_utils, and not directly in the top folder.

Reply:
Sure